src/main/java/duke/Commands.java

```diff
@@ -8,18 +8,34 @@
 import java.time.format.DateTimeFormatter;
 import java.util.Date;
 
+/**
+ * Represents a class that contains the logic for all keywords that Duke
+ * recognizes.
+ */
 public class Commands {
 
     public Commands() { // Empty Constructor
     }
 
+    /**
+     *  Prints a response to "bye" command to indicate that the command was
+     *  accepted.
+     */
     public void bye() { // Get DukeLCH to Exit
         String bye = "_______________________________________________________\n"
                 + "Goodbye. I hope to be of service to you again soon!\n"
                 + "_______________________________________________________\n";
         System.out.println(bye);
     }
 
+    /**
+     * A method that takes in a String, checks it against 2 accepted date formats
+     * and converts it into a default date format for output. If the String
+     * satisfies neither format, it is simply outputted as is.
+     *
+     * @param date A String input that represents a date.
+     * @return Either a default formatted date or the user input.
+     */
```

Comment:
Really love your detailed comments and how it clearly explains the use of each functions!:)